### PR TITLE
🎠 GITLAB_PROJECT was wrong  🎠

### PR DESCRIPTION
**Pull request by eformat**

### CODE REVIEW

File: exercises/2-attack-of-the-pipelines/README.md [hunk 1]
```diff
@@ -567,7 +567,7 @@ Some of the key things to note:
         // Global Vars
         NAMESPACE_PREFIX="<YOUR_NAME>"
         GITLAB_DOMAIN = "<GITLAB_FQDN>"
-        GITLAB_PROJECT = "<GIT_USERNAME>"
+        GITLAB_PROJECT = "<GITLAB_PROJECT>"
```
Comment (oybed) [inline exercises/2-attack-of-the-pipelines/README.md]: Actually, I think this is correct as-is .... although we could probably rename `GITLAB_PROJECT` to `GITLAB_USERNAME` ... i.e.: the environment variable `GITLAB_PROJECT` is used to compose the url for where the username is.

Comment (eformat) [reply exercises/2-attack-of-the-pipelines/README.md]: yeah, true that - is should be GITLAB_USERNAME i think, i will update this

**Comment by oybed** [comment]
LGTM although we should merge this and https://github.com/rht-labs/todolist/pull/1 at the same time. Looks like @eformat has requested @springdo to review that one.

**Comment by eformat** [comment]
> LGTM

feel free to merge both if @springdo not around, cheers

**Pull request merged**